woocommerce/woocommerce
# Add Jetpack Changelogger

### All Submissions:






### Changes proposed in this Pull Request:

Add [Jetpack Changelogger](https://github.com/Automattic/jetpack/tree/master/projects/packages/changelogger). This PR adds the changelogger capability for each project in the repo, but leaves the current processes intact until we are ready by using a `NEXT_CHANGELOG.md` file temporarily. More details in https://github.com/woocommerce/woocommerce/issues/31135.

### How to test the changes in this Pull Request:

1. `cd plugins/woocommerce`.
2. `composer install`.
3. `./vendor/bin/changelogger add`.
4. Follow the prompts to add a changelog.
6. `./vendor/bin/changelogger write --use-version=6.0`. This step will add your changelogs to `NEXT_CHANGELOG.md` along with the appropriate version bump.
7. Check `NEXT_CHANGELOG.md` to make sure changes are appropriately made.
8. Repeat the same process for any of the packages.

### Other information:




### Changelog entry

> Add Jetpack Changelogger

### FOR PR REVIEWER ONLY:
